src/corner_kick/src/style/base.css

```diff
@@ -1,24 +0,0 @@
-/* 
- * Defines the base styling for the application. Imports all other css files in this
- * directory.
- */
-
-@import url('https://fonts.googleapis.com/css?family=Open+Sans');
```

Comment:
similairly to above comment, this seems out-of-scope, I'm chill with it, but would like a quick explanation (maybe in the PR description?) about what we changed?

Reply:
Added to PR description